Package.swift

```diff
@@ -109,10 +109,10 @@ var targets: [PackageDescription.Target] = [
             "DistributedActorsTestKit",
         ],
         exclude: [
-          "DocumentationProtos/",
+            "DocumentationProtos/",
         ],
         plugins: [
-            "DistributedActorsGeneratorPlugin"
+            "DistributedActorsGeneratorPlugin",
```

Comment:
@ktoso Are trailing comas fine?

Reply:
I believe there should be an option to disable this if needed

Reply:
Yeah that's fine, looking good thanks